front-end/src/components/navigation-icon/NavigationIcon.scss

```diff
@@ -0,0 +1,17 @@
+.navigationIcon {
+    text-align: center;
+    font-size: 12px;
```

Comment:
All sizes (typography, or spacing) should be based off a system. Can you potentially use bootstrap typographic helpers for this? Avoid defining font-size everywhere.

My strategy is that you create a typographic reusable component, then start reusing them everywhere. That way, the UI will be consistent.

Reply:
Agreed. One way, is to define a sass variables file and define all typography, space etc. rules as constants and use them that way.